.github/workflows/ubuntu.yml

```diff
@@ -156,17 +154,17 @@ jobs:
       - name: archive-files
```

Comment:
Some of the `make` calls use the -j argument. I'd use -j2 for all of them since there are usually two CPUs available. (Can be double-checked with `python3 -c "import multiprocessing as m; print('CPUs:', m.cpu_count())"`)